src/pages/[platform]/build-a-backend/auth/connect-your-frontend/manage-user-sessions/index.mdx

```diff
@@ -318,7 +318,7 @@ Amplify.Auth.fetchAuthSession(options: .forceRefresh())
 ```
 </InlineFilter>
 
-## Next steps
-
-- [Learn how to access and validate user sessions server-side](/[platform]/build-a-backend/auth/connect-your-frontend/server-side/#session-validation)
+{/* ## Next steps */}
+{/* https://github.com/aws-amplify/docs/issues/7355 */}
```

Comment:
Why are we adding a link to GH exactly?

Reply:
I'm working on the server-side docs and this is easy to ctrl+f the codebase and find stale references to the page without removing the content. I can remove it, though